File: 1448-maximum-69-number/maximum-69-number.java
class Solution {
    public int maximum69Number (int num) {
        StringBuilder temp = new StringBuilder(String.valueOf(num));

        for (int i = 0, n = temp.length(); i < n; i++)  {
            if (temp.charAt(i) == '6') {
                temp.setCharAt(i, '9');
                return Integer.valueOf(temp.toString());
            }
        }

        return num;
    }
}

File: 1448-maximum-69-number/maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        string num_s = to_string(num);

        for (int i = 0; num_s[i] != '\0'; i++) {
            if (num_s[i] == '6') {
                num_s[i] = '9';
                return stoi(num_s);
            }
        }

        return stoi(num_s);
    }
};